dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/user/MeController.java

```diff
@@ -164,7 +164,7 @@
         Sets.newHashSet( UserSettingKey.values() ) );
 
     @GetMapping
-    public ResponseEntity<JsonNode> getCurrentUser( @CurrentUser( required = true ) User user,
+    public @ResponseBody ResponseEntity<JsonNode> getCurrentUser( @CurrentUser( required = true ) User user,
```

Comment:
If you have the `ResponseEntity` wrapper you do not need the `@ResponseBody` - but for any other return type it is needed.